code/modules/crafting/tailoring.dm

```diff
@@ -167,6 +167,29 @@
 	reqs = list(/obj/item/clothing/glasses/hud/skills/goggles = 1)
 	category = CAT_CLOTHING
 
+/datum/crafting_recipe/hudsunskills
+	name = "Skills HUD sunglasses"
+	result = list(/obj/item/clothing/glasses/hud/skills/sunglasses)
+	time = 2 SECONDS
+	tools = list(TOOL_SCREWDRIVER, TOOL_WIRECUTTER)
+	reqs = list(/obj/item/clothing/glasses/hud/skills = 1,
+				/obj/item/clothing/glasses/sunglasses = 1,
+				/obj/item/stack/cable_coil = 5)
+	category = CAT_CLOTHING
+
+/datum/crafting_recipe/hudsunskills/New()
+	..()
+	blacklist = subtypesof(/obj/item/clothing/glasses/hud/skills) \
+		| typesof(/obj/item/clothing/glasses/sunglasses) - ALLOWED_INGREDIENT_SUNGLASSES
```

Comment:
this is a bit gory, I would either just keep this all in one line or make it multiline (as in multiple operations) for readability

Reply:
All other hud recipes are written this way, I just copied it as is.

Reply:
fair